src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Configuration/IConfiguration.cs

```diff
@@ -91,6 +91,7 @@ public interface IConfiguration
         IEnumerable<string> ExpectedErrorClassesForAgentSettings { get; }
         IDictionary<string, IEnumerable<string>> ExpectedErrorMessagesForAgentSettings { get; }
         string ExpectedErrorStatusCodesForAgentSettings { get; }
+        IDictionary<string, IEnumerable<string>> ExpectedErrorsInfo { get; }
```

Comment:
What do you think about renaming this to `ExpectedErrors` or `ExpectedErrorConfiguration`